docs/content/mqtt.md

```
@@ -0,0 +1,165 @@
+**Example: Manually Locking Your Door via MQTT**
+
+Let's say you want to lock your door using a command-line MQTT client like `mosquitto_pub`.
+
+```bash
+mosquitto_pub -h your_mqtt_broker_address -p 1883 -t homekit/set_state -m "1"
+```
+(Replace `your_mqtt_broker_address` with your actual broker's IP or hostname.)
+
+**Example: Subscribing to Lock State Updates**
+
+To see your lock's current state in real-time:
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)